docs/docs/free-response-markets.md

```diff
@@ -0,0 +1,41 @@
+# Guide to Free Response markets
+
+# Overview
+
+In free-response markets, the creator asks an open ended question. Both the creator and users can propose answers which can be bet on. The payout system rewards users who bet on good answers early.
+
+# Basic facts
+
+- Markets are structured around a list of answers, any of which can be bet on.
+- Anyone can add answers to a market as long as they stake some amount of M$ on it.
+- Traders can place a bet on any answer and receive shares in the outcome in return.
+- When a user places a bet, their M$ goes into the pool and they receive a certain amount of shares of the selected answer.
+- When the market is resolved, you will be paid out according to your shares. If you own shares of answer #1, and the creator resolves to answer #1, the entire pool is divided up amongst the users who bet on answer #1 proportional to their shares.
+- The creator of each market is responsible for resolving each market. They can resolve to any single answer, or even multiple answers.
+  - Resolving to multiple answers allows the creator to choose a percentage for each selected answer (or distribute equally). The payout for any answer is taken from the amount of the total pool allocated to that answer.
+    - For example, let's take a free-response market with many answers. The pool for this market is $500, and you own 100 out of 500 total shares of answer #1.
+    - If the creator resolves to answer #1 only, you will receive `M$500 * (100 / 500) = M$100`.
+    - If the creator resolves 50% to answer #1 and 50% to answer #2, you will receive `(M$500 * 50%) * (100 / 500) = M$50`.
+    - Note that your payout is dependent on the total number of shares, and thus may decrease if more people buy shares in that answer.
+  - Creators can also resolve N/A to cancel all transactions and reverse all transactions made on the market - this includes profits from selling shares.
+
+# Market creation
```

Comment:
It seems clear that rather than repeating this stuff between the "free response" and "binary" market documents, we should reorganize stuff somehow. Same thing for the fee structure below.

Reply:
Good idea! Just merged those files and condensed it a tad.